SAGA/hosts/host_catalog.py

```diff
@@ -57,6 +53,11 @@ def __init__(self, database):
             self._host_name_to_id = dict(zip((n.lower() for n in t['SAGA']), t['NSA']))
             self._host_id_to_name = dict(zip(t['NSA'], t['SAGA']))
 
+        try:
```

Comment:
Any particular reason to move this section of code?

Reply:
Yes - it's because the `load` call ends up triggering `_annotate_catalog`, which requires that the ``'hosts_named'`` part of the database be loaded.  So the ``self.load()`` call has to come *after* the block that starts with ``self._database['hosts_named'].read()``.  As far as I could tell the order *otherwise* didn't matter, so this seemed the easy solution? (as opposed to some more complicated lazy-loading approach or something like that)

Reply:
sounds good